refugeerouter/models.py

```diff
@@ -73,17 +73,36 @@ class Flat(models.Model):
     is_barrier_free = models.BooleanField(default=True)
     notes = models.CharField(max_length=1024, blank=True)
 
+    def __str__(self):
+        return f'{self.address} (owned by {self.owner_last_name})'
+
 
 class Booking(models.Model):
+    BOOKING_STATE_CLOSED = 'C'
+    BOOKING_STATE_CONFIRMED = 'C'
+    BOOKING_STATE_DENIED = 'D'
+    BOOKING_STATE_INHABITED = 'I'
+    BOOKING_STATE_OPEN = 'O'
+    BOOKING_STATE_REQUESTED = 'I'
+    BOOKING_STATE_CHOICES = [
+        (BOOKING_STATE_OPEN, 'Open'),
+    ]
```

Comment:
this seems off?

Reply:
Off?

Reply:
Oh yes. Can you pls fix and then merge. Busy today :(